include/extractor/scripting_environment.hpp

```diff
@@ -36,6 +41,13 @@ class RestrictionParser;
 struct ExtractionNode;
 struct ExtractionWay;
 
+using index_sparse_type = osmium::index::map::SparseFileArray<osmium::unsigned_object_id_type, osmium::Location>;
+using index_dense_type = osmium::index::map::DenseFileArray<osmium::unsigned_object_id_type, osmium::Location>;
```

Comment:
Is there a reason you want the cache to live on disk? I would use these instead:

```c++
using SparseIndex = osmium::index::map::SparseMemArray<osmium::unsigned_object_id_type, osmium::Location>;
using DenseIndex = osmium::index::map::DenseMemArray<osmium::unsigned_object_id_type, osmium::Location>;
```

See https://github.com/Project-OSRM/osrm-backend/pull/3223#discussion_r86320537